### PR TITLE
Milestone1: First Page Layout

### PR DESCRIPTION
**Pull request by BertrandConxy**
## Created Home Page for Portfolio Project
- Created index.html file for markup contents and style.css for styling
- Used FlexBox for page layout which is mobile version skeleton.
- Build toolbar
- And the Headline section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+      <div class="nav">
```
Comment (rustam-tolipov) [inline index.html]: - [ ] The container of the section should be `<header>` tag. Change `<div>` into `<header>` and problem will be solved 👨‍🔧. Using semantic html elements makes your code better like: `<header>`, `<aside>`, `<time>` etc. 😃👌

File: style.css [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+/* Toolbox */
+
+.nav {
+  display: flex;
+  flex-flow: column wrap;
+  height: 50px;
+  align-content: space-between;
+}
```
Comment (rustam-tolipov) [inline style.css]: - [ ] Add some margin here like: `padding: 10px 12px` as given in your figma design. Using margin and paddings improves UX of project and also helps clients to understand your content easier 😃🙌

File: index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            <a href="#" class="logo"><h1>Welcome</h1></a>
+        </header>
+        <nav>  
+            <a href="#"><img  src="images/Icons/Menu.png" alt="togglebutton" class="MenuIcons"></a>
+        </nav>
+      </div> 
+      <main>
+        <section class="section-1">
+             
+                <h2 id="Primary-1">Hey there. I’m Mariane</h2>
```
Comment (rustam-tolipov) [inline index.html]: - [ ] Change `<h2>` to `<h1>` 🍀. Use h1 for important things. With doing that you will improve SEO, for example, if someone search your website they also will be able to see your name 🔍.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @BertrandConxy, 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BertrandConxy** [comment]
Thanks, @Rustamxon. I appreciate your reviews.
## Made the following changes:
-Used best Html and CSS practices for header tag as stated in the first review
-Added some padding to the Header tag, Welcome logo, and Menu button for best UX design
-Changed h2 tag to h1 used for the name so as to improve SEO.

I look forward to another best review.

**Review by keneogu — APPROVED** [review]
Hi @BertrandConxy,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Pull request merged**